python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -5,19 +5,29 @@
 from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection
 from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
 
-def is_relevant_syst_for_shape_corr(flavor_btv, syst):
+def is_relevant_syst_for_shape_corr(flavor_btv, syst, self):
```

Comment:
self is a special word in python. This is a bad choice of variable name

Reply:
Agreed, you can just pass era directly here, its the only way "self" is uesd

Reply:
Agreed was already working on making the change.